spec/models/schedule_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Schedule, type: :model do
+  describe "on create" do
+    it "raises error if there is no user" do
+      schedule = Schedule.new(name: "new schedule", start_date: Date.today,
+        end_date: Date.today + 1.week)
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.